parcellab/common/templates/_datadog.tpl

```diff
@@ -0,0 +1,36 @@
+{{/* vim: set filetype=mustache: */}}
+{{/*
+  Create chart/subchart name and version as used by the chart/subchart label:
+  {{ include "common.chart" (
+    dict
+      "Chart" "The Chart scope"
+      "Release" "The Release scope"
+      "Values" "The Values scope"
```

Comment:
I would document here that we pass `datadog` in the dict. I see that we call it in `.datadog`

Reply:
sure, copy & paste leftover